tuneme/templates/polls/detail.html

```diff
@@ -14,6 +14,7 @@
 	<h3>{{ question.title }}</h3>
 	{% if error_message %}<p><strong>{{ error_message }}</strong></p>{% endif %}
 		<form action="{% url 'molo.polls:vote' question.id %}" method="post">
+			{% if not user_has_voted %}
```

Comment:
I'm not sold on this - I think we need to find a way to prevent the user from needing to vote at all instead of only showing them an error once they voted. The way we've done it in the past is to have a method on the Poll that says `can_voter(user)` and returns whether or not the user is allowed to vote.